wordpress-mobile/WordPress-iOS
# [All Time Widget] Add functionality

Fixes #13030

This adds full functionality to the All-Time widget. (And thus completes the #13030 effort.)

To test:

---
- Start with a fresh install, or be logged out of the app.
- Add the `WordPress All-Time` widget to your Today view.
  - Verify the unconfigured view is displayed.
  - Verify the widget cannot be expanded.
  - Verify tapping the widget launches the app.

<img width="400" alt="unconfigured" src="https://user-images.githubusercontent.com/1816888/70265153-0adbec00-1757-11ea-859a-94fccadcb096.png">

---
- Log into the app.
- Go to Stats > Widgets > `Use this site`.
  - Verify the All-Time widget displays data for the selected site, and matches what is displayed in the app.
  - Verify the widget can be expanded/collapsed.

<img width="400" alt="app" src="https://user-images.githubusercontent.com/1816888/70265196-221ad980-1757-11ea-8b3d-6fd3e6042c0a.png">

<img width="400" alt="widget" src="https://user-images.githubusercontent.com/1816888/70266278-66a77480-1759-11ea-9767-5c76695e829a.png">

<img width="400" alt="widget_collapsed" src="https://user-images.githubusercontent.com/1816888/70266290-6c9d5580-1759-11ea-9373-543b889c19ce.png">

---
- Return to the app, and select a different site.
- Go to Stats > Widgets > `Use this site`.
- Verify the All-Time widget updates accordingly.

---
PR submission checklist:
